tests/spec/fs.rename.spec.js

```diff
@@ -10,6 +10,39 @@ describe('fs.rename', function() {
     expect(fs.rename).to.be.a('function');
   });
 
+  it('should not change an existing file to itself', function(done) {
+    var complete1 = false;
+    var complete2 = false;
+    var fs = util.fs();
+
+    function maybeDone() {
+      if(complete1 && complete2) {
+        done();
+      }
+    }
+
+    fs.open('/myfile', 'w+', function(error, fd) {
+      if(error) throw error;
+
+      fs.close(fd, function(error) {
+        if(error) throw error;
+
+        fs.rename('/myfile', '/myfile', function(error) {
+          if(error) throw error;
```

Comment:
The tests are failing because of this. 

Shouldn't the error be expected in this instance? the test is renaming `/myfile` to `/myfile`. So if that's the case, shouldn't the test expect an error rather then fail if an error occurs?